src/vs/workbench/browser/parts/statusbar/statusbarItem.ts

```diff
@@ -26,19 +26,23 @@ import { IHoverDelegate } from 'vs/base/browser/ui/iconLabel/iconHoverDelegate';
 
 export class StatusbarEntryItem extends Disposable {
 
-	readonly labelContainer: HTMLElement;
 	private readonly label: StatusBarCodiconLabel;
 
 	private entry: IStatusbarEntry | undefined = undefined;
-	get name(): string { return assertIsDefined(this.entry).name; }
 
 	private readonly foregroundListener = this._register(new MutableDisposable());
 	private readonly backgroundListener = this._register(new MutableDisposable());
 
 	private readonly commandMouseListener = this._register(new MutableDisposable());
 	private readonly commandKeyboardListener = this._register(new MutableDisposable());
 
-	private readonly hover = this._register(new MutableDisposable<ICustomHover>());
+	private hover: ICustomHover | undefined = undefined;
```

Comment:
@aeschli since we now update an existing hover if present, I decided to drop the `MutableDisposable`.